go.mod

```diff
@@ -4,6 +4,8 @@ go 1.17
 
 replace github.com/fluxcd/source-controller/api => ./api
 
+replace helm.sh/helm/v3 => helm.sh/helm/v3 v3.9.0-rc.1.0.20220627184942-c52cd40acef1
```

Comment:
Please add a comment and explain why is this needed, link to your Helm PR. Also add a TODO so we remove this once the change is included in the next Helm patch release.

Reply:
yes.